spec/event_tag_utils_spec.rb

```diff
@@ -72,5 +78,67 @@
       }
       expect(Optimizely::Helpers::EventTagUtils.get_revenue_value(event_tags)).to eq(9223372036854775807)
     end
+
+  end
+
+  describe '.get_event_value' do
```

Comment:
There are some test cases missing here, Can you please compare with the python SDK. 

e.g. isNan, maxfloat, minfloat, infinity, overflow. all these cases look missing to me.

Reply:
Please add expect(Optimizely::Helpers::EventTagUtils.get_revenue_value(true)).to be_nil in 
it 'should return nil if argument is not a Hash' like here
https://github.com/optimizely/python-sdk/blob/master/tests/helpers_tests/test_event_tag_utils.py#L30

Reply:
All cases are added simmilar to Python SDK,
# Ruby SDK
```
expect(Optimizely::Helpers::EventTagUtils.get_event_value(nil)).to be_nil
expect(Optimizely::Helpers::EventTagUtils.get_event_value(0.5)).to be_nil
expect(Optimizely::Helpers::EventTagUtils.get_event_value(65536)).to be_nil
expect(Optimizely::Helpers::EventTagUtils.get_event_value(9223372036854775807)).to be_nil
expect(Optimizely::Helpers::EventTagUtils.get_event_value('65536')).to be_nil
expect(Optimizely::Helpers::EventTagUtils.get_event_value(false)).to be_nil
expect(Optimizely::Helpers::EventTagUtils.get_event_value([])).to be_nil
```
there are no unit cases for maxfloat, minfloat, infinity, overflow in python either

Reply:
Check under this method

https://github.com/optimizely/python-sdk/blob/master/tests/helpers_tests/test_event_tag_utils.py#L82

Reply:
@oakbani go ahead add those